proxy/main_profiling.go

```diff
@@ -61,5 +62,5 @@ func main() {
 		}()
 	}
 
-	launchProxy(true)
+	launchProxy(true, configFile)
```

Comment:
did you try to compile this? I don't think this builds. I think we should put the configfile flag on a separate file that always builds regardless of whether profiling mode is enabled or not.